aten/src/ATen/native/SegmentReduce.cpp

```diff
@@ -80,5 +60,49 @@ Tensor _segment_reduce_cpu(
   return output;
 }
 
+} // namespace
+
+enum SegmentReductionType { MAX };
+static const std::map<std::string, SegmentReductionType> segmentReduce2REDUCE =
+    {
+        {"max", MAX},
+};
+
+Tensor _segment_reduce_(
+    const Tensor& data,
+    std::string reduce,
```

Comment:
const

Reply:
I tried to fix this but then the binding was not working properly, it complains with "undefined symbol"